next/README.md

```diff
@@ -0,0 +1,31 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
Add the line here also 
"for the scope for this example, we will focus on app router in nextjs. Please refer to [here](https://github.com/imagekit/imagekit-docs/-/blob/dev/page_router_example_in_sdk_repo) for page router"

Reply:
Done.